test/NinjaTools.FluentMockServer.Tests/Xunit/MockServerTestBase.cs

```diff
@@ -0,0 +1,33 @@
+using System;
+using System.Net.Http;
+using NinjaTools.FluentMockServer.Tests.TestHelpers;
+using NinjaTools.FluentMockServer.Xunit;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace NinjaTools.FluentMockServer.Tests.Xunit
+{
+
+    // [CollectionDefinition(nameof(MockServerCollectionFixture), DisableParallelization = true)]
+    // public class MockServerCollectionFixture : ICollectionFixture<MockServerFixture>
+    // { }
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/manual/git_36/NinjaTools.FluentMockServer/pullRequest?prid=4927362)